include/seqan3/alignment/pairwise/detail/pairwise_alignment_algorithm_banded.hpp

```diff
@@ -108,7 +108,8 @@ class pairwise_alignment_algorithm_banded :
                                          this->optimal_score,
                                          this->optimal_coordinate,
                                          alignment_matrix,
-                                         callback);
+                                         callback,
+                                         this->transfer_log());
```

Comment:
Here it is with `this->`

Reply:
Ah because `banded` uses unbanded as base?

Reply:
yes. In the one case I need to use this-> because it uses the base class feature.